### PR TITLE
inline css-resets in react docs and compositions

### PR DESCRIPTION
**Pull request by KutnerUri**
## Proposed Changes

- remove `css-resets` from compositions and docs
- inline css resets in documenter components
- improve composition cards (center without applying flex, max width, prevent leakage)
- improve spacing in documenter components.
- move documentation components to documenter
- inlineCode changed from accent-color to gray

### TODO
- [x] tag documenter components, and bump version in bit-bin

### CODE REVIEW

File: .bitmap [hunk 1]
```diff
@@ -865,30 +865,12 @@
...
-        "version": "0.0.397",
-        "mainFile": "index.ts",
-        "rootDir": "scopes/mdx/ui/create-heading"
-    },
-    "ui/docs/highlighted-text-span": {
-        "scope": "teambit.mdx",
-        "version": "0.0.397",
-        "mainFile": "index.ts",
-        "rootDir": "scopes/mdx/ui/highlighted-text-span"
-    },
```
Comment (KutnerUri) [inline .bitmap:879]: moved these 3 components to documenter scope

File: scopes/react/ui/docs-app/base.module.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 .docsMainBlock {
-  padding: 50px 40px 0 40px;
+  padding: 50px 40px 10px 40px;
```
Comment (KutnerUri) [inline scopes/react/ui/docs-app/base.module.scss:2]: fixes margin collapsing at the bottom of the page

File: scopes/react/ui/docs-app/base.module.scss [hunk 3]
```diff
@@ -42,26 +42,11 @@
...
-    &[align='left'] {
-      text-align: start;
-    }
-    &[align='right'] {
-      text-align: end;
-    }
-  }
-  > * {
-    margin-bottom: 16px;
-  }
```
Comment (KutnerUri) [inline scopes/react/ui/docs-app/base.module.scss:66]: moved to documenter

File: scopes/compositions/compositions/ui/compositions-overview/compositions-overview.tsx [hunk 4]
```diff
@@ -6,14 +6,15 @@ import styles from './compositions-overview.module.scss';
...
 };
 
-export function CompositionsOverview({ compositions }: CompositionsOverviewProps) {
+export function CompositionsOverview({ compositions, compositionCardClass }: CompositionsOverviewProps) {
   return (
     <div className={styles.background}>
       {compositions &&
         Object.entries(compositions).map(([key, composition]) => (
-          <CompositionCard key={key} Composition={composition} name={key} />
+          <CompositionCard key={key} Composition={composition} name={key} className={compositionCardClass} />
```
Comment (odedre) [inline scopes/compositions/compositions/ui/compositions-overview/compositions-overview.tsx:17]: dont we want this anymore?

File: scopes/compositions/ui/composition-card/composition-card.module.scss [hunk 5]
```diff
@@ -18,15 +18,18 @@
...
+  margin: auto; // assumes container is flex
   padding: 16px;
   box-sizing: border-box;
 
-  overflow: hidden;
+  // prevent leakage:
+  overflow: auto;
+  position: relative; // for position: absolute elements
+  transform: translateZ(0); // prevents elements with `position: fixed` (etc) from leaking out of the composition card
+  max-width: 100%;
```
Comment (odedre) [inline scopes/compositions/ui/composition-card/composition-card.module.scss:32]: did you check this in ripple as well?

File: scopes/compositions/ui/composition-card/composition-card.tsx [hunk 6]
```diff
@@ -10,19 +10,19 @@ import { Icon } from '@teambit/evangelist.elements.icon';
 import { themedText } from '@teambit/base-ui.text.themed-text';
 import styles from './composition-card.module.scss';
 
-export type CompositionCardProps = {
+export interface CompositionCardProps extends CardProps {
```
Comment (odedre) [inline scopes/compositions/ui/composition-card/composition-card.tsx:13]: dont we need to omit the `elevation` ?

**Review by odedre — APPROVED** [review]

File: scopes/compositions/ui/composition-card/composition-card.tsx (hunk 6 above)
Comment (KutnerUri) [reply scopes/compositions/ui/composition-card/composition-card.tsx:13]: can override if needed using `...rest`

**Pull request merged**